src/components/Header/SearchBar/SearchBar.component.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useRef} from  'react';
+
+import styled from 'styled-components';
+
+const SearchBox = styled.div`
+    background-color:grey;
+    align-self: center;
+    margin-left: 1em;
+    margin-right: 1em;
+    height: 2em;
+    border: none;
+    flex-direction: row;
+    border-radius: 0.5em;
+`;
+
+const Input = styled.input `
+    border: none;
+    margin-top: 0.5em;
+    background-color: grey;
+    font-size: 0.9em;
+    color: white;
+    outline: none;
+    .focus{
+        color: white;
+    }
+    ::placeholder {
+        color: white;
+    }
+`;
+
+
+const SearchBar = (props) => {
+    const [search, setSearch] = useState("")
+    const searchInput = useRef(null)
+
+    const handleFocus = () => {
+        searchInput.current.focus()
+    }
+    
+    const sendData = (evt) => {
+        setSearch(evt.target.value);
+        props.callBack(evt.target.value);
+    }
+    return (
+        <>
```

Comment:
Since the root component is `SearchBox` there is no need for the fragment here. You should be good to remove it.